ops/instrument_pam.go

```diff
@@ -102,10 +103,20 @@ func PAMTracingHandler(tracerName string, attributes ...attribute.KeyValue) pipe
 	}
 }
 
-// getRequestName, return "GetSession" from "GetBalanceRequest"
+// getRequestName, return "vplugin.PluginPAM/GetBalance" from "GetBalanceRequest"
 func getRequestName(req any) string {
-	name, _ := strings.CutSuffix(reflect.TypeOf(req).Name(), "Request")
-	return name
+	var name string
+
+	t := reflect.TypeOf(req)
+	if t.Kind() == reflect.Pointer {
+		name = t.Elem().Name()
+	} else {
+		name = t.Name()
+	}
+
+	name, _ = strings.CutSuffix(name, "Request")
+
+	return fmt.Sprintf("%s/%s", RPCService, name)
```

Comment:
Isn't this method used both when using genericpam and vplugin. in the `PAMMetricHandler`. Adding "vplugin.PluginPAM" doesn't seem correct in the generic case

Reply:
Right you are again! 😆 Let me fix that!